mapswipe_workers/tests/unittests/tear_down.py

```diff
@@ -0,0 +1,43 @@
+"""Helper functions for test tear down"""
+
+from mapswipe_workers import auth
+
+
+def delete_test_data(project_id: str) -> None:
+    """
+    Delete test project indluding groups, tasks and results
+    from Firebase and Postgres
+    """
+    fb_db = auth.firebaseDB()
+    ref = fb_db.reference("v2/results/{0}".format(project_id))
+    ref.delete()
+    ref = fb_db.reference("v2/tasks/{0}".format(project_id))
```

Comment:
depending on the size of your project this step might not work due to `writeSizeLimit`. (e.g. see in delete_project function of mapswipe_workers.)

Reply:
Should work in all cases though, because the test data is small (Only a subset of tasks and groups will be uploaded to firebase)